distributor-api-v1/operations.md

```diff
@@ -19,7 +19,6 @@ Preferred initial call used to obtain all static data about distributor configur
 
 |  | Property | Type | Description |
 | :--- | :--- | :--- | :--- |
-| `Client` | string | required | Identification of the client as described in [Authorization](https://mews-systems.gitbook.io/distributor-guide/distributor-api-v1/authorization). |
```

Comment:
Those clients have to stay in the table, because they are presented in the example of the request.